lib/Optimize.cpp

```diff
@@ -170,7 +171,7 @@ void OptimizeModule(const EntityLifter &lifter, llvm::Module &module) {
 
   fpm.addPass(llvm::SCCPPass());
   fpm.addPass(llvm::DSEPass());
-  fpm.addPass(llvm::SROA());
+  fpm.addPass(llvm::SROAPass());
```

Comment:
Looks like this is an incompatible change. Are we happy for me to `ifdef` around this to support both 13 and 14?

Reply:
@pgoodman mentioned that we'd be dropping support for older versions at some point. Do we want to do this now and only support 14+ going forwards?

Reply:
[ec5ab11](https://github.com/lifting-bits/anvill/pull/297/commits/ec5ab11ab20bf33b0c20cc9afabe89ac0a36f174)